core/controller/src/main/scala/whisk/core/controller/API.scala

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
i suspect so - wonder if we should leave the file name unchanged for now to make a diff easier.